src/pages/index.js

```diff
@@ -70,31 +127,113 @@ function createCard(data) {
   const cardElement = new Card(
     { data },
     selectors.cardTemplate,
-    handleImageClick
+    handleImageClick,
+    handleDeleteClick,
+    handleLikeClick,
+    handleDislikeClick
   );
   return cardElement.getView();
 }
 
 /*------- Event Handlers --------*/
 
-function handleImageClick(imgData) {
-  imageModal.open(imgData);
+function handleImageClick(title, image) {
+  imageModal.open(title, image);
 }
 
-function handleProfileFormSubmit(formValues) {
-  updateUserInfo(formValues);
+function handleProfileFormSubmit(inputs) {
+  const newName = inputs.name;
+  const newDescription = inputs.description;
+  profileEdit.renderSaving(true);
+  api
+    .updateUserInfo(newName, newDescription)
+    .then((response) => {
+      console.log("Profile updated successfully:", response);
+      currentUserInfo.setUserInfo({
+        name: newName,
+        description: newDescription,
+      });
+
+      profileEdit.close();
+    })
+    .catch((error) => {
+      console.error("Error updating profile:", error);
+    });
+  profileEdit.renderSaving(false);
```

Comment:
`  profileEdit.renderSaving(false);
`

You need to return the **default** button text only in blocks `finally` so that the button could get the text back in any case: `success` or `failure`.


And this way you'll see the loading effect


**Please, correct it everywhere**